pkg/openstack/loadbalancer.go

```diff
@@ -2268,6 +2303,26 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(ctx context.Context, clusterName
 	return mc.ObserveReconcile(err)
 }
 
+func (lbaas *LbaasV2) deleteFIPIfCreatedByProvider(fip *floatingips.FloatingIP, portID string, service *corev1.Service) (bool, error) {
+	matched, err := regexp.Match("Floating IP for Kubernetes external service", []byte(fip.Description))
+	if err != nil {
+		return false, err
```

Comment:
curious how this can be a error :) 

if it's error ,should we just return `false, nil` ?

Reply:
Yeah, my mistake.

Reply:
I basically moved it from [1] making no changes. The only error can come from Compile of the regex, so I guess it's deterministic. If you want I can make it ignore the error.

[1] https://github.com/kubernetes/cloud-provider-openstack/blob/2231e343644690f3e672c024af89d695fe20ac3c/pkg/openstack/loadbalancer.go#L2336-L2339

Reply:
But I think I'd rather return the `err`, as when it happens it means we haven't properly processed the FIP and we shouldn't make the workflow to progress.

Reply:
just curious the check here, I think we can take it without change